### ISSUE TITLE
App rejected with passcode lock

### ISSUE THREAD

**Issue by alexwinston** [yankodimitrov/SwiftPasscodeLock#79]
This might not technically be considered a bug but I wanted to enquire as to whether anyone has had this issue.  I just received a rejection from Apple for the following reason

"Your app contains features that mimic the passcode iOS interface ."

I am not sure why they are clamping down on this now but has anyone else run into this?  I have read a few people using similar lock views have to change the buttons to not be round and maybe add a logo.

Thoughts?

**Comment by Bhide** [comment]
Yes. My app recently got rejected for this, but only the recent version of the app because we posted a screenshot of the screen on iTunes Connect. The earlier version passed the review as we didn't had the screenshot attached on iTunes Connect.

I've also added the logo but still... 

I guess changing the button structure to anything other than the rounded one should be allowed :)

**Comment by klwoon** [comment]
@Bhide Is your app approved after changing the buttons shape?

**Comment by Bhide** [comment]
@klwoon Yes. It got approved after I changed the button shape.
